tests/integration/conftest.py

```diff
@@ -7,7 +9,7 @@
 def vcr_config():
     # this fixture is called by the pytest-recording vcr decorator.
     return {
-        "record_mode": "new_episodes",
+        "record_mode": "none" if os.getenv("CI") else "new_episodes",
```

Comment:
great we needed that.
It will still succeed if the person changed the prompt.
Because I missed the match_on attribute.
I am currently adding it back BUT test_browse_website is flaky, meaning it keeps recording new HTTP requests.
So let's add this in, but keep in mind it will still pass in the pipeline.

the benefit of this change is that once we add the matchers, it will say that the cassette doesn't match. Without this change it would have attempted to query open ai with a dummy api key